onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -125,18 +125,62 @@ def aten_adaptive_max_pool3d_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::avg_pool2d", trace_only=True)
 def aten_avg_pool2d(
-    self: TensorType,
+    self: TFloat,
     kernel_size: Sequence[int],
```

Comment:
We may need to use a wrangler to change the np arrays back to a list, if `kernel_size`, `stride` and `padding` are supplied as np.arrays. I suspect this was why there was an assertion error in the full graph test - we passed them as inputs mistakenly because they were preprocessed as np arrays (since they are supplied as positional arguments).

We will soon be able to distinguish between inputs and arguments even better after the OpSchema PR (https://github.com/onnx/onnx/pull/5020) in onnx is merged.